docker/instant-seal-node.dockerfile

```diff
@@ -36,9 +36,6 @@ ENTRYPOINT ["/frequency/frequency", \
 	# Required params for starting the chain
 	"--dev", \
 	"-lruntime=debug", \
-	"--instant-sealing", \
```

Comment:
Question, do we have to remove this? I ask because of the latency of when I will be able to see a potential issue versus when it "happens" e.g. instant-sealing keeps things nice and neat and doesn't create a crap load of blocks, Matter of being able to come in in the morning and just being able to look at the latest block MeWe had an issue with (something like this)

Reply:
It is removed from the entrypoint, but added back into the `CMD`. So it still works that way by default, but now it can be overridden if you want the docker image to do something different!